src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,13 @@
 package core.basesyntax;
 
 public class MainApp {
+    private static final Machine[] machines = new Machine[]
```

Comment:
There is not reason to make this field constant, so please make it local method variable